connection.php

```diff
@@ -0,0 +1,725 @@
+<?php
+
+/**
+* @author: mfsi-krishnadev
+* @class : database connection and operation
+* 
+*/
+class db_connection 
+{
+	//declaration of variables
+	private $hostname = "localhost";
+	private $username = "root";
+	private $password = "mindfire";
+	private $database = "employee";
+	public  $connect;
+
+	//contructor to initialse the variable
+	function __construct()
+	{
+		$this->connect = mysqli_connect($this->hostname,$this->username,$this->password,$this->database);
+
+		//Error Handling
+		if(mysqli_connect_error())
```

Comment:
Use spaces consistently.
some places there is no space after if like at this line and then many place there is a space after it.
so either use space or not use space and be consistent.
Better to use space for better good looking code.
Check all places and fix it.